Sources/Swift/SentryExperimentalOptions.swift

```diff
@@ -1,7 +1,18 @@
 @objcMembers
 public class SentryExperimentalOptions: NSObject {
+    /**
+     * Disables swizzling of`NSData` to automatically track file operations.
+     *
+     * - Note: Swizzling is enabled by setting ``SentryOptions.enableSwizzling`` to `true`.
+     *         This option allows you to disable swizzling for `NSData` only, while keeping swizzling enabled for other classes.
+     *         This is useful if you want to use manual tracing for file operations.
+     */
+    public var disableDataSwizzling = false
```

Comment:
`h`: I just saw that I approved the PR for adding the option `enableFileManagerSwizzling` in GH-4634. As our other options related to auto-tracing don't contain swizzling, I suggest renaming `disableDataSwizzling` to `enableDataTracing` and `enableFileManagerSwizzling` to `enableFileManagerTracing`. As `enableFileManagerSwizzling` is experimental I think we don't need to deprecate it. We can simply rename it with pointing it out in the changelog.

Reply:
I added a comment to the PR so we can find it again in the future. Please check it, leave feedback and review again.